pipelines/batch/setup_prod_job.py

```diff
@@ -167,6 +174,7 @@ def main(
                 report_date="latest",
                 n_warmup=n_warmup,
                 n_samples=n_samples,
+                output_dir=str(Path("output", output_subdir)),
```

Comment:
I have the same concerns about output here as in `setup_eval_job.py`

Reply:
See response above.